Documentation/helm-values.rst

```diff
@@ -736,6 +736,10 @@
      - Configure how frequently garbage collection should occur for the datapath connection tracking table.
      - string
      - ``"0s"``
+   * - :spelling:ignore:`conntrackGCMaxInterval`
+     - Configure how the maximum frequency for the garbage collection of the connection tracking table. Only affects the automatic computation for the frequency and has no effect when 'conntrackGCInterval' is set. This can be set to more frequently clean up unused identities created from ToFQDN policies.
+     - string
+     - ``"0s"``
```

Comment:
Looks like this description is outdated. Can you update this file? (ie `cd Documentation && make update-helm-values`)

Reply:
Oops, always forget one of these steps.